examples/tour.jl

```diff
@@ -76,8 +76,8 @@ for s in [Tables.Schema((:a, :b, :c, :d), (Real, String, Any, AbstractVector)),
     @test validate(s, FooV1SchemaVersion()) isa Nothing
 
     # if `find_violations` finds any violations, it returns a vector of tuples indicating the relevant
-    # fields and their violations; returns `nothing` otherwise
-    @test isnothing(find_violations(s, FooV1SchemaVersion()))
+    # fields and their violations
+    @test isempty(find_violations(s, FooV1SchemaVersion()))
```

Comment:
https://github.com/beacon-biosignals/Legolas.jl/pull/85#discussion_r1169236443

these should still be `nothing`

Reply:
> these should still be `nothing`

Why though? `find_violations` is a new function added in #85 so we can have it use an empty array instead of `nothing` which seems better. As long as we don't modify `find_violation`'s behavior nothing is broken

Reply:
> > these should still be `nothing`
> 
> Why though? `find_violations` is a new function added in #85 so we can have it return an empty array instead of `nothing` which seems better. As long as we don't modify `find_violation`'s behavior nothing is broken